templates/startup/entrypoint_scheduler.sh

```diff
@@ -2,6 +2,19 @@ echo "Starting up airflow scheduler"
 # Sanity check the dags
 ls /opt/airflow/dags
 
+sudo apt update
```

Comment:
Not pretty, but how I'm installing dependencies when exec'ing into the container